common/tasks/rosa-hcp/hcp-cluster-test-pipeline.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: hcp-cluster-test-pipeline
+spec:
+  description: "This pipeline creates a ROSA HCP cluster, runs tests, and then deletes the cluster."
+  params:
+    - name: ocp-version
+    - name: region
+    - name: hcp-secrets
+  tasks:
+    - name: generate-cluster-name
+      taskRef:
+        resolver: git
+        params:
+          - name: url
+            value: https://github.com/xinredhat/konflux-qe-definitions.git
+          - name: revision
+            value: hcp_cluster
+          - name: pathInRepo
+            value: common/tasks/rosa-hcp/generate-cluster-name.yaml
+    - name: provision-rosa-cluster
+      runAfter:
+        - generate-cluster-name
+      taskRef:
+        resolver: git
+        params:
+          - name: url
+            value: https://github.com/xinredhat/konflux-qe-definitions.git
+          - name: revision
+            value: hcp_cluster
+          - name: pathInRepo
+            value: common/tasks/rosa-hcp/rosa-hcp-provision.yaml
+      params:
+        - name: cluster-name
+          value: "$(tasks.generate-cluster-name.results.cluster-name)"
+        - name: ocp-version
+          value: "$(params.ocp-version)"
+        - name: region
+          value: "$(params.region)"
+        - name: node_number
+          value: "4"
+        - name: hcp-secrets
+          value: "$(params.hcp-secrets)"
+    - name: test
+      runAfter:
+        - provision-rosa-cluster
+      params:
+        - name: ocp-login-command
+          value: "$(tasks.provision-rosa-cluster.results.ocp-login-command)"
+      taskSpec:
+        params:
+          - name: ocp-login-command
+            description: The command to login to the created cluster
+        steps:
+          - name: run-tests
+            image: quay.io/konflux-qe-incubator/konflux-qe-tools:latest
+            script: |
+              set -o errexit
+              set -o nounset
+              set -o pipefail
+              echo "Running tests..."
+              echo "$(params.ocp-login-command)"
+    - name: delete-rosa-cluster
+      runAfter:
+        - test
+      taskRef:
+        resolver: git
+        params:
+          - name: url
+            value: https://github.com/xinredhat/konflux-qe-definitions.git
```

Comment:
Dont use your fork values

Reply:
good catch, I forgot to update it after verfication

Reply:
fixed.